js/logo.js

```diff
@@ -4345,6 +4345,22 @@ function Logo () {
                 that._setListener(turtle, listenerName, __listener);
             }
             break;
+		case 'noise':
+			if (args[0] == null) {
+                that.errorMsg(NOINPUTERRORMSG, blk);
+                childFlow = args[1];
+                childFlowCount = 1;
+			} else if (args[0] == "white" || args[0] == "pink" || args[0] == "brown") {
```

Comment:
You need to also test for the _() versions of the string to support translation.